include/uk/event.h

```diff
@@ -105,9 +105,9 @@
  *       return UK_EVENT_HANDLED;
  * }
  *
- * UK_EVENT_HANDLER_PRIO(handler1, UK_PRIO_EARLIEST);
- * UK_EVENT_HANDLER_PRIO(handler2, UK_PRIO_LATEST);
- * UK_EVENT_HANDLER_PRIO(handler3, UK_PRIO_LATEST);
+ * UK_EVENT_HANDLER_PRIO(event, handler1, UK_PRIO_EARLIEST);
+ * UK_EVENT_HANDLER_PRIO(event, handler2, UK_PRIO_LATEST);
+ * UK_EVENT_HANDLER_PRIO(event, handler3, UK_PRIO_LATEST);
```

Comment:
Might as well go ahead and also mention in the comments how to trigger an event, I think this is also missing

```c
        struct myevent_data data;
        // ...
        int ret = uk_raise_event(myevent, &data);
```

Reply:
That's already there https://github.com/unikraft/unikraft/blob/ac15f671186430848629313836715616687ff4e5/include/uk/event.h#L79

Reply:
whoops, missed it, closing this